runtime/src/epoch_stakes.rs

```diff
@@ -39,6 +101,7 @@ impl VersionedEpochStakes {
             total_stake,
             node_id_to_vote_accounts: Arc::new(node_id_to_vote_accounts),
             epoch_authorized_voters: Arc::new(epoch_authorized_voters),
+            bls_pubkey_to_rank_map: OnceLock::new(),
```

Comment:
The code looks good to me. However, before we have BLS Pubkey landed in Vote accounts this map will never be initialized.